packages/owlbot-bootstrapper/cli/helper.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {CliArgs} from './run-trigger-command';
+import {execSync} from 'child_process';
+
+export class Helper {
+  args: CliArgs;
+  languageContainers: {
+    language: string;
+    languageContainer: string;
+    repoToClone: string;
+  }[];
+
+  constructor(args: CliArgs) {
```

Comment:
I would not import this type from the CLI (it couples the 2 pieces of code), instead declare the type within this module. If the types match, then you can just pass the parsed args object to this constructor.

Reply:
This helper is in the cli though, these would have to be coupled regardless (it would be the same type). What if we instead created an interfaces file that exported the type to the various files that needed it in cli/?

Reply:
The logic class should define the interface and its caller should use that interface. The caller (CLI) could also declare its own type and if it matches the logic class' interface then that's fine. Or alternatively, if the shared interface is truly shared (something that makes sense like something that actually is a concept in the domain), then it could be declared separately and shared.

Reply:
In this case, the only things this class needs are the `language` and `languageContainer` fields. So we should declare only those arguments. You can still pass the entire `CliArgs` object to this constructor.